ui/components/src/ui-components/propertygrid/component/VirtualizedPropertyGrid.tsx

```diff
@@ -15,6 +15,7 @@ import { areEqual, ListChildComponentProps, VariableSizeList } from "react-windo
 import { assert } from "@bentley/bentleyjs-core";
 import { PropertyRecord } from "@bentley/ui-abstract";
 import { Orientation, RatioChangeResult } from "@bentley/ui-core";
+import { MutableCategorizedPrimitiveProperty } from "../../../ui-components";
```

Comment:
Should not import from barrel file of the same repo

Reply:
Changed